api/api.py

```diff
@@ -215,6 +215,9 @@ def put_me(user):
     if 'stall_description' in request.json:
         user.stall_description = bleach.clean(request.json['stall_description'])
 
+    if 'nostr_private_key' in request.json:
```

Comment:
we should NOT allow users to change the private key!

Reply:
This is needed so the call to `putProfile` works.

Reply:
Ah, right, we are storing the PM-generated key, not the other key. True, sorry...